internal/pkg/print/print.go

```diff
@@ -197,6 +197,27 @@ func (p *Printer) PagerDisplay(content string) error {
 	return nil
 }
 
+func (p *Printer) FileOutput(outputFileName, content string) error {
+	fo, err := os.Create(outputFileName)
+	if err != nil {
+		return fmt.Errorf("create output file: %w", err)
+	}
+
+	defer func() {
+		tempErr := fo.Close()
+		if tempErr != nil {
+			err = fmt.Errorf("close output file: %w", tempErr)
+		}
+	}()
+
+	_, err = fo.WriteString(content)
+	if err != nil {
+		return fmt.Errorf("write content to output file: %w", err)
+	}
+
+	return nil
+}
```

Comment:
I don't think this should be part of the `print` package since it doesn't use printer functionality. We should have a `fileutils` package to do things like this. In the `feature/multi-config` branch, we have something like this implemented, although inside the `config` package for some reason:
https://github.com/stackitcloud/stackit-cli/blob/807d6887c422d96ece8d412b5f7491aac96b7f60/internal/pkg/config/file_utils.go#L11

I think you should extract this function to `pkg/file_utils/file_utils.go` and then I'll move the code in the `multi-config` branch to there as well

Reply:
This function as is will never return that error as it just assigns it to a variable that is not used afterwards.

In this case if we also want to capture the error of closing the file we would need to use a named parameter ad do something like:

```suggestion
func (p *Printer) FileOutput(outputFileName, content string) (err error) { // Notice the named variable here
	fo, err := os.Create(outputFileName)
	if err != nil {
		return fmt.Errorf("create output file: %w", err)
	}

	defer func() {
		tempErr := fo.Close()
		if tempErr != nil {
            if err != nil {
				err = fmt.Errorf("%v; close output file: %w", err, tmpErr)
			} else {
				err = fmt.Errorf("close output file: %w", tmpErr)
			}
		}
	}()

	_, err = fo.WriteString(content)
	if err != nil {
		return fmt.Errorf("write content to output file: %w", err)
	}

	return err
}
```

Reply:
Regarding your point @DiogoFerrao: also tend to agree it would fir better in another package, as I see it no as an output of the CLI but as part of the functionality of the command to write something to a file. For this reason also would rename it to something like `WriteToFile`